Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -3192,7 +3192,7 @@ extension BrowserViewController: WKUIDelegate {
     }
 
     let linkPreview: UIContextMenuContentPreviewProvider? = { [unowned self] in
-      if let tab = tabManager.tabForWebView(webView) {
+      if let tab = tabManager.tabForWebView(webView), url.schemeIsValid, ["http", "https"].contains(url.scheme) {
```

Comment:
fyi we have `isWebPage(includeDataURIs:)` in `URLExtensions` if you want to use that